lib/cog/repo.ex

```diff
@@ -1,3 +1,13 @@
 defmodule Cog.Repo do
   use Ecto.Repo, otp_app: :cog
+
+  alias Cog.Queries.Repo, as: Queries
+
+  def exists?(model, id, opts \\ [true: true, false: false]) do
```

Comment:
Is this true false thing used anywhere yet? If not, I'd opt to remove it and just return true/false.